spec/lib/api/redfish-1.0/system-spec.js

```diff
@@ -771,6 +783,10 @@ describe('Redfish Systems Root', function () {
     });
 
     it('should return a valid system', function() {
+        redfish.getVendorNameById.resolves({
+            node: node,
+            vendor: 'notDellAndCisco'
```

Comment:
It's purpose is to get the results of the third system information which is not dell and Cisco